src/components/application_manager/test/hmi_capabilities_test.cc

```diff
@@ -1132,18 +1132,18 @@ TEST_F(HMICapabilitiesTest,
 }
 
 TEST_F(HMICapabilitiesTest,
-       SaveCachedCapabilitiesToFile_CacheFileNameNotSpecified_NoNeedToSave) {
+       SaveCachedCapabilitiesToFile_CacheFileNameNotSpecified_ReturnLikeSuccess) {
```

Comment:
@ydementieiev I propose 
CacheFileNameNotSpecified_NoNeedToSave_ReturnSuccess

Reply:
@LitvinenkoIra ok thank you , I will rename in next commit

Reply:
@LitvinenkoIra  [done](https://github.com/smartdevicelink/sdl_core/pull/3300/commits/06763373047e2c89482f757571f7710655f28a66)